examples/tv-casting-app/tv-casting-common/include/CHIPProjectAppConfig.h

```diff
@@ -58,6 +58,9 @@
 #define CHIP_CONFIG_EXAMPLE_ACCESS_CONTROL_MAX_SUBJECTS_PER_ENTRY 20
 #define CHIP_CONFIG_EXAMPLE_ACCESS_CONTROL_MAX_ENTRIES_PER_FABRIC 20
 
+// For casting, we need to allow for more binding table entries
+#define MATTER_BINDING_TABLE_SIZE 64
```

Comment:
this change is not described in the PR summary: why is it tied to the other changes?

Reply:
Was looking at the tile ... the change is in the summary.

However could you please also describe the _why_ the change is done? The summary says "I changed it" and the comment here says "we need more" however it does not say why more or now. It may be things like "this runs on large devices, so having more bindings is possible and better" or "we discovered this use case that needs more bindings because it is common" or something like that.

Readers should understand why changes are done and see if assumptions change over time (i.e. can I make this larger or smaller over time? If I at some point decide to make it smaller again, what do I break?)

Reply:
I've updated the config file and the PR summary with the reasons why this is required.